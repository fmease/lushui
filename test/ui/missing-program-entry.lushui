# TEST compiler-args --component-type=executable

