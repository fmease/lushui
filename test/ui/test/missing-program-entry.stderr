error[E050]: the component ‘missing-program-entry’ does not contain a ‘main’ function in its root module
 --> ${TEST_FOLDER}/missing-program-entry.lushui:1:1
  |
1 |   ;;; TEST compiler-args --component-type=executable
  |  _^
2 | | 
  | |^
  |

error: aborting due to previous error

